frontend/packages/integration-tests-cypress/tests/app/overview.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { Set as ImmutableSet } from 'immutable';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+
+import { nav } from '../../views/nav';
+import * as yamlEditor from '../../views/yaml-editor';
+import { overviewPage } from '../../views/overview';
+import { detailsPage } from '../../views/details-page';
+import { errorMessage } from '../../views/form';
+import { masthead } from '../../views/masthead';
+
+import { checkErrors, testName } from '../../support';
+import {
+  DeploymentModel,
+  StatefulSetModel,
+  DeploymentConfigModel,
+  DaemonSetModel,
+} from '../../../../public/models';
+
+const overviewResources = ImmutableSet([
+  DaemonSetModel,
+  DeploymentModel,
+  DeploymentConfigModel,
+  StatefulSetModel,
+]);
+
+describe('Visiting Overview page', () => {
+  afterEach(() => {
+    checkErrors();
+  });
+
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.switcher.changePerspectiveTo('Administrator');
+    nav.sidenav.switcher.shouldHaveText('Administrator');
+    cy.createProject(testName);
+  });
+
+  overviewResources.forEach((kindModel) => {
+    describe(kindModel.labelPlural, () => {
+      const resourceName = `${testName}-${kindModel.kind.toLowerCase()}`;
+
+      before(() => {
+        cy.visit(`k8s/ns/${testName}/${kindModel.plural}/~new`);
+        masthead.username.shouldBeVisible();
+        yamlEditor.isLoaded();
+        yamlEditor.getEditorContent().then((content) => {
+          const newContent = _.defaultsDeep(
+            {},
+            { metadata: { name: resourceName, labels: { automatedTestName: testName } } },
+            safeLoad(content),
+          );
+          yamlEditor.setEditorContent(safeDump(newContent)).then(() => {
+            yamlEditor.clickSaveCreateButton();
+            cy.get(errorMessage).should('not.exist');
+            detailsPage.titleShouldContain(resourceName);
+            cy.visit(`/k8s/cluster/projects/${testName}/workloads`);
+          });
+        });
+      });
+
+      it(`displays a ${kindModel.id} in the overview list page`, () => {
+        overviewPage.projectOverviewIsVisible();
+        overviewPage.itemsAreVisible();
+        overviewPage.groupLabelItemPresent(kindModel);
+        overviewPage.projectOverviewListItemPresent(resourceName);
+      });
+
+      /* Disabling for now due to flake https://jira.coreos.com/browse/CONSOLE-1298
+      xit(`CONSOLE-1298 - shows ${kindModel.id} details sidebar when item is clicked`, () => {
```

Comment:
Hi, so [CONSOLE-1298](https://issues.redhat.com/browse/CONSOLE-1298) is "Test flake: Visiting Overview shows details sidebar when item is clicked".   I recommend recreating this test in Cypress and see if it is still an issue.  Cypress in general has better 'waiting' conditions and it would be good to verify this behavior when clicking on an item.

Reply:
@dtaylor113 done!